src/functions/calculateTargetAllocation.ts

```diff
@@ -1,55 +1,74 @@
 import Allocation, { AssetTypes } from "src/interfaces/allocation"
+import StableValueTokensAPI from "src/interfaces/stable-value-tokens"
 
 const EMPTY_TARGETS: Allocation[] = [
   { type: "celo-native-asset" as const, token: "CELO", percent: 0 },
   { type: "other-crypto-assets" as const, token: "BTC", percent: 0 },
   { type: "other-crypto-assets" as const, token: "ETH", percent: 0 },
   { type: "stable-value" as const, token: "DAI", percent: 0 },
   { type: "stable-value" as const, token: "USDC", percent: 0 },
+  { type: "stable-value-eur" as const, token: "EUROC", percent: 0 },
   { type: "natural-capital" as const, token: "cMCO2", percent: 0 },
 ]
 
 export function calculateTargetAllocation(
```

Comment:
I think we could break it down to make it more readable. However, since we're planning a site revamp, I'm not sure how valuable it woule be. I'll let you decide.